samples/bookinfo/platform/kube/rbac/rbac-permissive-telemetry.yaml

```diff
@@ -0,0 +1,45 @@
+# Configuration for logentry instances
+apiVersion: "config.istio.io/v1alpha2"
+kind: logentry
+metadata:
+  name: newlog
```

Comment:
can we use more specific name. Here and others

Reply:
updated.